build/git/testsuite.log

```diff
@@ -1,2830 +1,108 @@
-Manifying 9 pod documents
 *** t0000-basic.sh ***
-ok 1 - .git/objects should be empty after git init in an empty repo
```

Comment:
I would prefer to leave the individual test lines in so that we can see which tests changed their result between package versions.